Library/Homebrew/os/mac/xcode.rb

```diff
@@ -139,7 +139,8 @@ def uncached_version
       end
 
       def provides_autotools?
-        version < "4.3"
+        # Xcode 2.5's autotools are too old to rely on at this point
+        (version < "4.3") && (version > "2.5")
```

Comment:
Sorry, I mean that 4.3 is old enough that I wonder if this method could just be `false` without really causing much grief?